recipes-extended/baseboxd-tools/baseboxd-tools/bundle-debug-info

```diff
@@ -334,6 +343,10 @@ if [ $((COLLECT_MASK & COLLECT_MSTPD)) -gt 0 ]; then
   get_mstpd_state
 fi
 
+if [ $((COLLECT_MASK & COLLECT_SWITCH)) -gt 0 ]; then
+  get_switch_state
```

Comment:
The function call here does not match the function name. One of them needs to change.

Reply:
Oops, missed this one. Fixed.